build_data.js

```diff
@@ -351,14 +351,17 @@ function generateTranslations(fields, presets, tstrings) {
         var tags = p.tags || {};
         var keys = Object.keys(tags);
 
+        var tagsDescription = keys.map(function(k) { return k + '=' + tags[k]; }).join(', ');
+
         if (keys.length) {
-            preset['name#'] = keys.map(function(k) { return k + '=' + tags[k]; }).join(', ');
+            preset['name#'] = `Name for '${tagsDescription}'. Aliases may be added after the name, separated by commas.`;
         }
         if (p.searchable !== false) {
+            preset['terms#'] = `Related terms beside the name and aliases with which the preset for '${tagsDescription}' should be found, separated by commas. '<>' if none are eligible.`;
             if (p.terms && p.terms.length) {
-                preset['terms#'] = 'terms: ' + p.terms.join();
+                preset['terms#'] += ' English terms: ' + p.terms.join();
             }
-            preset.terms = '<translate with synonyms or related terms for \'' + preset.name + '\', separated by commas>';
+            preset.terms = '<>';
         } else {
```

Comment:
The `....#` stuff is the description that is shown in Transifex as "Developer Notes".

The description for `name` mentions the possibility to add aliases after the name.
The description for `terms` now also mentions what they are used for and how to explicitly omit them. The previous message directly in the translation was sometimes misunderstood by translators so that they literally translated the English terms.

cc @1ec5